docs/results/array_slice_with_negative_step_only.html

```diff
@@ -322,6 +315,13 @@ <h4 id="Rust_jsonpath_lib">
 $[::-2]
 ^^^^
 </code></pre>
+
+<h4 id="dotNET_JsonPathLib">
+dotNET (JsonPathLib)
+</h4>
+
+<pre><code>timeout: sending signal TERM to command 'build/Dotnet_JsonPathLib'
+</code></pre>
```

Comment:
This might be a flaky run. The library is sometimes slow to respond, we could consider upping the timeout value.

For now you can force a re-run with

    rm build/results/array_slice_with_negative_step_only/dotNET_JsonPathLib
    ./src/wrap_in_docker.sh ninja